test/components/entity/list.spec.js

```diff
@@ -1039,13 +1035,12 @@ describe('EntityList', () => {
         });
     });
 
-    it('disables filters and download button', () => {
+    it('disables filters', () => {
       testData.extendedEntities.createPast(1, { label: 'My Entity' });
       testData.extendedEntities.createPast(1, { label: 'deleted 1', deletedAt: new Date().toISOString() });
       testData.extendedEntities.createPast(1, { label: 'deleted 2', deletedAt: new Date().toISOString() });
       return loadEntityList({ props: { deleted: true } })
         .afterResponse(component => {
-          component.find('#entity-download-button').attributes('aria-disabled').should.equal('true');
```

Comment:
since this got removed, is there another test of the download button being disabled for deleted entities? i can't find it.